ChangeLog

```diff
@@ -1,4 +1,26 @@
-* Tue Apr 28 2020 Klaus Wenninger <klaus.wenninger@aon.at> Pacemaker-2.0.4
+* Wed May 13 2020 Klaus Wenninger <klaus.wenninger@aon.at> Pacemaker-2.0.4-rc2
+- Changesets: 28
+- Diff: 28 files changed, 437 insertions(+), 130 deletions(-)
+
+- Changes since Pacemaker-2.0.4-rc1
+  + controller: properly detect remote node info requests
```

Comment:
Regression since 2.0.0 not 2.0.3?
The points I had marked with 'regression rc1' are actually everything I would remove when finally merging into a 2.0.4 ChangeLog entry (real regressions since 2.0.3 respectively stuff that was new since 2.0.3 and had to be fixed).

Reply:
Yes, this one line is a real regression since 2.0.0

The rest can be '(regression since 2.0.4-rc1)'

Reply:
Regression introduced since 2.0.0 or with 2.0.0 (meaning 2.0.0 was buggy already)?

Reply:
taking back my above question - full log says it

Reply:
I hadn't thought about that before, but I guess "since" is ambiguous. I use it to mean "in specified release and all subsequent releases before this one".

Reply:
"introduced in" works for me